common/scripted_score_values/wc_scripted_score_values.txt

```diff
@@ -212,4 +212,23 @@ stereotypical_traits_game_rule_score = {
 			value = more_100
 		}
 	}
+}
+
+killing_speed_score = {
+	modifier = {
+		factor = 0.01
```

Comment:
Is this the value that modifies the plot's speed? I could play around with it to see the difference more clearly.

Reply:
Yes. As I know, this is the smallest factor that can be put. `0.001` won't work.